spec/models/category_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Category', type: :model do
+  before(:each) do
+    @user = User.create(name: 'Victoria')
+    @subject = Category.create(name: 'Cat test', icon: 'Icon test', user_id: @user.id)
+  end
+
+  it 'Should be created successfully' do
+    expect(@user.name).to eq('Victoria')
+  end
+
+  it 'name should not be valid' do
+    @subject.name = nil
+    expect(@subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] all your tests are failing please make sure to fix them so that they are passing 

<img width="1032" alt="Screen Shot 2022-10-14 at 18 20 05" src="https://user-images.githubusercontent.com/61790587/195906924-3f4921bb-9ed1-4296-bb82-5ef732f4b3d3.png">